unit-tests/src/org/commcare/android/tests/queries/CaseDbQueryTest.java

```diff
@@ -66,7 +66,19 @@ public void testCaseIndexQueries() {
                 "test_case_id_child_2", ec);
     }
 
-    public static void evaluate(String xpath, String expectedValue, EvaluationContext ec) {
+    @Test
+    public void testCaseOptimizationTriggers() {
```

Comment:
Can we move this to test to `commcare-core`? It looks like it doesn't have anything Android specific